fractal/server/app/models/security.py

```diff
@@ -19,10 +19,18 @@
 """
 
 
+def new_uuid() -> UUID4:
+    # This should fix the (randomly-appearing) badly-formed UUID errors
```

Comment:
please add reference to relevant issue from upstream in docstring

Reply:
Added a reference to https://github.com/tiangolo/sqlmodel/issues/25#issuecomment-982039809 in the docstring.

Also note that we could just update sqlmodel to solve this problem without custom workarounds - see issue #147.